packages/eventemittable/__tests__/eventemittable.js

```diff
@@ -0,0 +1,197 @@
+var EventEmitter = require('../node_modules/events').EventEmitter;
```

Comment:
That was unexpected! `require('../node_modules/events')` :)
Have never seen that before :)